iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractVoltageLimitsTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.tck;
+
+import com.powsybl.iidm.network.Load;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.VoltageLimits;
+import com.powsybl.iidm.network.test.EurostagTutorialExample1Factory;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public abstract class AbstractVoltageLimitsTest {
+
+    private static Network createNetwork() {
```

Comment:
Finally we allow voltage limits on equipments ? I thought we decided to limit voltage limits to buses and bus bar sections. I am not sure to be able to simulate that level of details in the open load flow for example.